assets/assets/utils/chem_utils.py

```diff
@@ -0,0 +1,60 @@
+import os
+from typing import Union
+
+from indigo import Indigo
+from indigo.renderer import IndigoRenderer
+
+from assets import exceptions, logger
+
+SUPPORTED_FORMATS = {
+    ".pdf": "pdf",
+    ".mol": "chem",
+    ".sdf": "chem",
+    ".png": "image",
+    ".jpg": "image",
+    ".jpeg": "image",
+    ".txt": "text",
+}
```

Comment:
Could we make it, let's say, enum or named tuple? This way we won't compare hardcoded extensions in the code

Reply:
and values of the enum could be tuples or other named tuple with fields `extension` and `category`, where category is another enum.

Reply:
Can you please provide an example, how do you see this mapping can be done using NamedTuple?